src/toolboxes/spline/Canvas.tsx

```diff
@@ -105,7 +105,7 @@ export class SplineCanvas extends Component<Props> {
     return (
       <BaseCanvas
         onClick={this.onClick}
-        cursor={this.props.isActive ? "crosshair" : "none"}
+        cursor={this.props.isActive ? "crosshair" : "pointer"}
```

Comment:
Should this default to pointer? 

I think that will mean this canvas shows a cursor even when another tool is activated?

(eg when we draw a paintbrush cursor in the future it would have a pointer in?)

Reply:
Fair... will reset in my next PR